src/stylesheets/main.scss

```diff
@@ -20,6 +20,7 @@ $app-light-grey: #f8f8f8;
 
 body {
   margin: 0;
+  @include govuk-font-regular-19;
```

Comment:
Can we add this to the example component, rather than sticking it on the body? As I understand it that was the only thing that wasn't rendering correctly?